### PR TITLE
[BEAM-2141] Update jenkins job for JDBCIOIT

### PR DESCRIPTION
**Pull request by ssisk**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [X] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [X] Each commit in the pull request should have a meaningful subject line and body.
 - [X] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [X] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [X] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [X] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---
cc @jasonkuster

### CODE REVIEW

**Comment by ssisk** [comment]
run seed job

**Comment by ssisk** [comment]
retest this please

**Comment by ssisk** [comment]
a manual run was triggered of the job I added (https://builds.apache.org/view/Beam/job/beam_PerformanceTests_JDBC/171/) - it failed with 

[WARNING] Rule 2: org.apache.maven.plugins.enforcer.RequireMavenVersion failed with message:
Detected Maven Version: 3.0.5 is not in the allowed range [3.2,).

@jasonkuster speculated the problem might be that the jenkins job no longer does the setup steps for maven (ie, that they got removed with my changes), but I can't see how my changes would have affected that.


Looking further up in the file, I see this error:
[WARNING] Failed to inject repository session properties.
java.lang.NoSuchMethodError: org.apache.maven.execution.MavenSession.getRepositorySession()Lorg/eclipse/aether/RepositorySystemSession;
	at kr.motd.maven.os.RepositorySessionInjector.injectRepositorySession(RepositorySessionInjector.java:22)
	at kr.motd.maven.os.DetectExtension.injectSession(DetectExtension.java:151)

but I don't know if that's useful or not

**Comment by ssisk** [comment]
run seed job

**Comment by ssisk** [comment]
run seed job

**Comment by ssisk** [comment]
run seed job

File: .test-infra/jenkins/job_beam_PerformanceTests_JDBC.groovy [hunk 1]
```diff
@@ -32,32 +32,31 @@ job('beam_PerformanceTests_JDBC'){
...
+            '-e',
+            "-Pdataflow-runner",
+            'clean',
+            'install',
+            "-pl runners/google-cloud-dataflow-java",
+            '-DskipTests'
+              ]
 
-    common_job_properties.buildPerformanceTest(delegate, argMap)
+    io_it_suite_command = [
```
Comment (kennknowles) [inline .test-infra/jenkins/job_beam_PerformanceTests_JDBC.groovy]: This command is lacking a maven goal

Comment (ssisk) [inline .test-infra/jenkins/job_beam_PerformanceTests_JDBC.groovy]: thanks! fixed it.

**Comment by ssisk** [comment]
run seed job

**Comment by ssisk** [comment]
Issue 1
=====
last run was https://builds.apache.org/view/Beam/job/beam_PerformanceTests_JDBC/174/console - I believe it failed because kubernetes is not installed on the machine. It failed when invoking kubernetes, and I realized we haven't done anything to install kubectl on the jenkins machines.

Issue 2
=====
I also think there's another problem we haven't hit yet, which is that we'll need the full path to maven before we can invoke it - the mvn binary on the path on this machine isn't the correct one. 

I think we have a couple options here: 
1) pass maven_binary when we run pkb.py. That means modifying the maven poms to allow users to pass in the maven_binary flag there and then passing it on as a flag to pkb.py
2) make sure the version of mvn on the path is the correct one when running this jenkins job.
3) switch this job to a regular maven job, and figure out how to compile everything AND invoke io-it-suite. That could mean switching from the "use prebuilt repo" I'm currently doing in pkb

cc @jasonkuster since I'll need your help on this

**Comment by jasonkuster** [comment]
Issue 1 is correct. I'll chat with Infra to get kubernetes installed on the machines. We handle issue 2 already -- check out the 'maven_binary' arg we're passing. That path should remain stable, so you can use it with no worries.

**Comment by ssisk** [comment]
closing this pull request since Cham is working on a new version

**Pull request closed without merging**